components/button/stories/button.stories.js

```diff
@@ -58,6 +58,13 @@ export default {
 		isActive,
 		isPending,
 		staticColor,
+		noWrap: {
+			name: "Disable Label Wrap",
```

Comment:
```suggestion
			name: "Disable label wrap",
```
Use [sentence case](https://spectrum.adobe.com/page/grammar-and-mechanics/#Sentence-case) on this. That's something we've been working on making more consistent across the docs lately.